Readme.md

```diff
@@ -14,67 +14,67 @@ $ clib install clibs/buffer
 ## API
 
 ```c
-buffer_t *
-buffer_new();
+clib_buffer_t *
```

Comment:
the initial PR was just the fix for the non-c99 warnings -> see 7b6828a
the rest was done - because i could no integrate it in one of my existing projects (wich uses libssh) wich has a `buffer_t` typedef so i forked it - and replaced all buffer_ with clib_buffer_ just to be namesafe!